SourceX/miniwin/misc_msg.cpp

```diff
@@ -75,6 +75,7 @@ static int translate_sdl_key(SDL_Keysym key)
 		return DVL_VK_F9;
 	case SDLK_F10:
 		return DVL_VK_F10;
+#ifndef __AMIGA__

```

Comment:
Are these `#ifndef`s in this file needed? I would assume that the AMIGA would simply not emit these events?

Reply:
Should be SDL1

Reply:
Yeah, I think so. Are these keys used in Diablo? I think the ifdefs were there in case we had to remap them.

Reply:
@arczi84 They are only ifdef'd because those keys doesn't exist on Amiga. They do exist in SDL1.2

Reply:
Some are: https://github.com/diasurgical/devilutionX/blob/e88e574c5f1e537007bb634873ea90270d92d9fd/Source/diablo.cpp#L883

Reply:
If our SDL supports F11 and F12 we should leave it because V4 supports it.

Reply:
That alienates everyone else that uses the default Amiga keyboard though. And a lot of people will probably rig up an Amiga keyboard to their V4 as well. That is "BigGun"-talk. Euugh...

Reply:
I think F11 and F12 on v4  are used already to enable/disable scanlines, so not used on AmigaOS anyway.

Reply:
F11 and F12 are only used for quick pre-configured chat messages, so since you have NONET enabled I'm going to assume that players aren't going to miss it much. Only other keys that I noticed that have an ingame effect are:
Printscreen: (take a screen shot in PCX format)
Page Up/Down: Quick navigate the menu and shops.

Reply:
Alright, that's fine then. If someone really wants these features, they'll need to open up a PR on our fork. ^^